### ISSUE TITLE
Full Accessibility & 508 Office Audit Comparison Tool Caution Flags Enhancements

### ISSUE THREAD

**Issue by dneel-bah** [department-of-veterans-affairs/va.gov-team#8880]
## Steps to complete the Full Accessibility & 508 Audit: 
- [x] Product Manager: create issue and fill in feature name in the title and other bolded information appropriately
- [x] Practice area reviewers create sub-issues attached to this one with feedback for your team on *feature name*
- [x] **Requesting team** completes attached feedback tickets based on definition of done included in each practice area feedback ticket (may differ for each practice area, please refer to attached tickets for further instruction)
- [x] All requesting team attendees complete brief [VSP Collaboration Cycle Feedback](https://adhoc.optimalworkshop.com/questions/20260uu8-0-0/questions/before) survey

## Artifacts - _please provide the following artifacts when you request a full review_
Environments: 
    - Staging: https://staging.va.gov/gi-bill-comparison-tool/
    - PROD: https://www.va.gov/gi-bill-comparison-tool

Schools with Caution Flags:
    - Dodge City Community College
    - Devry
    - American National University

- Link to your accessibility test plan:
    - [Test Rails Test Plan](https://dsvavsp.testrail.io/index.php?/runs/view/282&group_by=cases:section_id&group_order=asc)

## Additional Notes
As part of this review, VSP will send your product to VA's 508 office for approval (as applicable)

The caution flag data loaded into GIDS, causes information to be displayed in 3 locations within the comparison tool:
    1. On a school info card on the Search Results Page displayed after completing a search on the Landing Page.
    2. At the top of the profile page after the user selects a school from the Search Results Page.
    3. Towards the bottom of the profile page in the Cautionary Information section.

**Comment by 1Copenut** [comment]
@dneel-bah I heard back from the VA 508 office RE the caution flags. Contact's quote below:

> I also reviewed the warning in the GI Bill Tool, I think it worked fine I didn’t notice any defects.

I'll take a look this week and let you know any issues I find, though I suspect there won't be any.

**Comment by dneel-bah** [comment]
@1Copenut Do you have any updates on this ticket?

**Comment by 1Copenut** [comment]
@dneel-bah Probably late today or mid-day tomorrow.

**Comment by 1Copenut** [comment]
@dneel-bah I only found one small issue. I've created an epic, attached it here, and put the issue inside that epic. This is a bit on the formal side for one issue, but helps me track the progress and keeps the standard cadence and procedure we're using on VSP.

I'll be out for a few days, but the information in the ticket should be enough to take action.

Please reach out to the VSP product support team in my absence if you have questions.

**Comment by dneel-bah** [comment]
All sub issues have been addressed and resolved; closing this issue out.
